src/tests/hack-details.test.js

```diff
@@ -1,21 +1,9 @@
 import React from "react";
 import { render, fireEvent } from "@testing-library/react";
-import HackDetails from "../components/hack-details";
+import { HackDetails } from "../components/hack-details";
 
-test("Title Rendered", () => {
-  const { getByText } = render(<HackDetails />);
-  const element = getByText(/Idea/);
-  expect(element).toBeInTheDocument();
+test("If a container exists for displayng the hacks", () => {
+  const { container } = render(<HackDetails />);
+  expect(container.firstChild).toHaveClass("MuiGrid-root");
```

Comment:
I'd prefer to use a data-testId than a material ui class name (we hav no control over whether this changes or no between versions of material ui).